core/pom.xml

```diff
@@ -4,15 +4,15 @@
   <parent>
     <groupId>org.pac4j.jax-rs</groupId>
     <artifactId>parent</artifactId>
-    <version>5.0.1-SNAPSHOT</version>
+    <version>6.0.0-SNAPSHOT</version>
```

Comment:
I went ahead and set this to the next major version. It seemed it was going to be too messy trying to keep both javax and jakarta artifacts.